certified-connectors/FlowForma V2/apiDefinition.swagger.json

```diff
@@ -0,0 +1,399 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "FlowForma V2",
```

Comment:
@FlowForma It is not allowed to create a new connector folder for the version of your existing connector. Please remove the postfix "V2" in the connector folder name and title value.

Reply:
What if any effect would this have on the old connector, that was published in 2017 using a different publishing process, but has the same title?
The current connector is a new connector. That's why we called it "FlowForma V2". What would be the correct way to ensure distinguishable "V2" in the title, as seen with existing connectors?

Reply:
@FlowForma If you look into the repo, you can see the updates applied to the existing connectors. And it is not required to add a version number to the title. Please remove it from the connector folder name and title.

https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title

Reply:
This is a completely different connector, that's why we called it "FlowForma V2". The old "FlowForma" connector will be renamed as deprecated. These 2 connectors use completely different API architectures. so the "V2" is not a postfix, but a part of the connector's name.